app/controllers/root_controller.rb

```diff
@@ -3,7 +3,27 @@
 class RootController < ApplicationController
   layout "design_system"
 
+  PERMITTED_FILTER_STATES = %w[
+    draft
+    amends_needed
+    in_review
+    fact_check
+    fact_check_received
+    ready
+    scheduled_for_publishing
+    published
+    archived
+  ].freeze
+
   def index
-    @presenter = FilteredEditionsPresenter.new
+    states_filter_params = filter_params.to_h[:states_filter]
+    sanitised_states_filter_params = states_filter_params&.select { |fp| PERMITTED_FILTER_STATES.include?(fp) }
```

Comment:
not sure if there is any sanitisation on states filter here, seems more like filtered states params

Reply:
The sanitisation is the removal of any states that aren't valid.